manim/scene/three_d_scene.py

```diff
@@ -210,31 +211,58 @@ def move_camera(
 
         """
         anims = []
-        value_tracker_pairs = [
-            (phi, self.renderer.camera.phi_tracker),
-            (theta, self.renderer.camera.theta_tracker),
-            (distance, self.renderer.camera.distance_tracker),
-            (gamma, self.renderer.camera.gamma_tracker),
-            (zoom, self.renderer.camera.zoom_tracker),
-        ]
-        for value, tracker in value_tracker_pairs:
-            if value is not None:
-                anims.append(ApplyMethod(tracker.set_value, value, **kwargs))
-        if frame_center is not None:
-            anims.append(
-                ApplyMethod(
-                    self.renderer.camera._frame_center.move_to, frame_center, **kwargs
-                ),
-            )
 
-        self.play(*anims + added_anims)
+        if config.renderer != "opengl":
+            from ..camera.three_d_camera import ThreeDCamera
+
+            self.camera: ThreeDCamera
+            value_tracker_pairs = [
+                (phi, self.camera.phi_tracker),
+                (theta, self.camera.theta_tracker),
+                (distance, self.camera.distance_tracker),
+                (gamma, self.camera.gamma_tracker),
+                (zoom, self.camera.zoom_tracker),
+            ]
+            for value, tracker in value_tracker_pairs:
+                if value is not None:
+                    anims.append(tracker.animate.set_value(value))
+            if frame_center is not None:
+                anims.append(self.camera._frame_center.animate.move_to(frame_center))
+        else:
+            from ..mobject.opengl_mobject import OpenGLMobject
+
+            if frame_center is not None:
+                if isinstance(frame_center, OpenGLMobject):
+                    frame_center = frame_center.get_center()
+                frame_center = list(frame_center)
+
+            cam_anim = ""
+            for value, method in [
+                [phi, "set_phi"],
+                [theta, "set_theta"],
+                [gamma, "set_gamma"],
+                [
+                    f"{config.frame_width / zoom}, 0" if zoom is not None else None,
+                    "rescale_to_fit",
+                ],
+                [frame_center, "move_to"],
+            ]:
+                if value is not None:
+                    cam_anim += f".{method}({value})"
+            exec(f"anims.append(self.camera.animate{cam_anim})")
+            if distance is not None:
+                raise NotImplementedError(
```

Comment:
Wondering if it would be better to just log a warning here rather than raise an exception? If it isn't supported in opengl and distance can be ignored here maybe we can still let it render?

Reply:
Yup, updated.